src/winetricks

```diff
@@ -12909,11 +12911,12 @@ load_vcrun2019()
             # 2020/10/03: 4b5890eb1aefdf8dfa3234b5032147eb90f050c5758a80901b201ae969780107
             # 2020/11/13: b1a32c71a6b7d5978904fb223763263ea5a7eb23b2c44a0d60e90d234ad99178
             # 2021/03/09: f299953673de262fefad9dd19bfbe6a5725a03ae733bebfec856f1306f79c9f7
+            # 2021/03/09: b6c82087a2c443db859fdbeaae7f46244d06c3f2a7f71c35e50358066253de52
```

Comment:
The date here is wrong. (copy & paste error)

Reply:
corrected...